docs/Reference/Limits.md

```diff
@@ -0,0 +1,42 @@
+# Evergreen Limits
+
+Evergreen has different kinds of limits in place. The ones that users are likely to encounter are listed here.
+
+## Non-expirable Hosts
```

Comment:
Preference to say here "Unexpirable" or "Never Expiring" to fit other docs ("Never Expire" is used in the UI so that could work too)